src/DocumentFragment.js

```diff
@@ -0,0 +1,38 @@
+  // ### DocumentFragment Shim
+  //
+  // A representation of a document fragment that can be normally generated
+  // by calling `document.createDocumentFragment`. Since the document fragment
+  // is itself a node, it inherits all properties of a node.
+module.exports = (function(){
+  "use strict";
+
+  var Node = require('./Node');
+
+  function DocumentFragment() {
+    Node.call(this, Node.DOCUMENT_FRAGMENT_NODE, "#document-fragment");
+  }
+  DocumentFragment.prototype = Object.create(Node.prototype);
```

Comment:
What is the compatibility matrix for slowparse?  `Object.create` isn't supported in all browsers, in case that matters here.  See http://kangax.github.io/compat-table/es5/#Object.create